runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsOverlay.java

```diff
@@ -127,9 +241,39 @@ else if (plugin.getRoomBlacklist().contains(room.getBoss().getName().toLowerCase
 						color = Color.RED;
 					}
 
+					String bossName = room.getBoss().getName();
```

Comment:
This entire workflow is wrong. In my opinion, you shouldn't even be using item images. But you should be using ImageComponent if you are.

Reply:
Changed item rendering code to use ImageComponent.